clients/browser/index.html

```diff
@@ -444,28 +518,62 @@
     // Safari quirks: don't use the same var name in global scope as id of html element
     var overlay_ = document.querySelector('#overlay');
 
-    Nimiq.init(function($) {
-        window.$ = $;
-        $.network.connect();
-
-        overlay_.style.display = 'none';
-        window.ui = new MinerUI($);
-    }, function(code) {
-        document.getElementById('blockchain-title').classList.remove('initializing');
-        switch (code) {
-            case Nimiq.ERR_WAIT:
-                overlay_.style.display = 'block';
-                break;
-            case Nimiq.ERR_UNSUPPORTED:
-                alert('Browser not supported');
-                break;
-            default:
-                alert('Nimiq initialization error');
-                break;
+    function copyWalletSeed() {
+        if (typeof $ === "undefined") {
+            return;
         }
-    });
 
-    document.getElementById('blockchain-title').classList.add('initializing');
+        var mainContent = document.getElementById('content');
+        var input = document.createElement("INPUT");
```

Comment:
In JavaScript code, we use `'` instead of `"` for strings. Also occurs at some other locations.